docs/deployment/oss-deployment.md

```diff
@@ -1,30 +1,231 @@
+
+## Configure your CI job
+
+The following sections describe methods to customize your CI job.
+
+<!-- 
+### Pass or fail the CI job
+
+By default, a Semgrep CI job exits with exit code 1 if the scan returns any findings. This causes the job to fail.
+
+Semgrep provides **fail open** options. These options enable you to suppress findings that block your pipeline:
+
+<dl>
+	<dt><code>semgrep scan</code></dt>
+	<dd><strong>Fail</strong> on blocking findings, but <strong>passes</strong> on internal errors. This is the default behavior.</dd>
+	<dt><code>semgrep scan --no-suppress-errors</code></dt>
+	<dd>The Semgrep CI job <strong>fails</strong> on blocking findings and on internal errors.</dd>
+	<dt><code>semgrep scan || true</code></dt>
+	<dd><strong>Pass</strong> on blocking findings and on internal errors.</dd>
+</dl>
+
+Refer to [Semgrep exit codes](/docs/cli-reference/#exit-codes) to understand various internal issues that cause Semgrep to fail.
+-->
+<!--
+### Diff-aware scanning
+
+<DiffAwareScanning />
+-->
+
+### Schedule your scans
+
+<CiScheduling />
+
+### Customize rules and rulesets
+
+#### Add rules to scan with `semgrep scan`
+
+You can customize what rules to run in your CI job. The rules and rulesets can come from the [Semgrep Registry](https://semgrep.dev/explore/), or your own rules. The sources for rules to scan with are:
+
+* The value of the `SEMGREP_RULES` environment variable.
+* The value passed after `--config`. You can use multiple `--config` arguments, one per value. For example: `semgrep scan --config p/default --config p/comment`.
+
+The `SEMGREP_RULES` environment variable accepts a list of local and remote rules and rulesets to run. The `SEMGREP_RULES` list is delimited by a space (` `) if the variable is exported from a shell command or script block. For example, see the following BitBucket Pipeline snippet:
+
+```yaml
+# ...
+  script:
+    - export SEMGREP_RULES="p/nginx p/ci no-exec.yml" 
+    - semgrep ci
+# ...
+```
+
+The line defining `SEMGREP_RULES` defines three different sources, delimited by a space:
+
+```
+- export SEMGREP_RULES="p/nginx p/ci no-exec.yml" 
+```
+
+The example references two rulesets from Semgrep Registry (`p/nginx` and `p/ci`) and a rule available in the repository (`no-exec.yml`).
+
+If the `SEMGREP_RULES` environment variable is defined from a YAML block, the list of rules and rulesets to run is delimited by a newline. See the following example of a GitLab CI/CD snippet:
+```YAML
+# ...
+variables:
+  SEMGREP_RULES: >-
+    p/nginx
+    p/ci
+    no-exec.yml
+# ...
+```
+
+#### Write your own rules
+
+Write custom rules to enforce your team's coding standards and security practices. Rules can be forked from existing community-written rules.
+
+See [Writing rules](/writing-rules/overview/) to learn how to write custom rules.
+
+### Ignore files
+
+<CiIgnoringFiles />
```

Comment:
I think the last link in this section that leads to the Ignoring findings page is broken

Reply:
It also talks about `semgrep ci`, so I just ended up removing the whole section :smile: